composer.json

```diff
@@ -12,15 +12,6 @@
     "drupal/config_split": "^2.0",
     "dpc-sdp/bay_platform_dependencies": "^2.0.2"
   },
-  "extra": {
-    "enable-patching": true,
-    "composer-exit-on-patch-failure": true,
-    "patches": {
-      "drupal/jwt": {
```

Comment:
I'm not sure here, but should the whole thing be deleted or just the actual patch?

Reply:
I did it by mistake. Fixed it.